src/accessories/CleanUpHelper.ttslua

```diff
@@ -248,21 +248,32 @@ function getTrauma()
   end
 
   -- get campaign log
-  local campaignLog = findObjects(6)[1]
+  campaignLog = findObjects(6)[1]
```

Comment:
What are these constants?

Reply:
- `campaignLog` is the object reference to the campaign log
- `findObjects` is a function that uses a cast to find an object (the first object from that search being the campaign log in this case)
- the 6 stands for the type of search to perform